examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -311,13 +286,20 @@ void AppTask::FunctionTimerEventHandler(AppEvent * aEvent)
     }
 }
 
-int AppTask::SoftwareUpdateConfirmationHandler(uint32_t offset, uint32_t size, void * arg)
+#ifdef CONFIG_MCUMGR_SMP_BT
+void AppTask::StartSMPBLEAdvertisingHandler(AppEvent * aEvent)
 {
-    // For now just print update progress and confirm data chunk without any additional checks.
-    LOG_INF("Software update progress %d B / %d B", offset, size);
+    GetDFUOverSMP().StartBLEAdvertising();
+}
 
-    return 0;
+void AppTask::RequestSMPAdvertisingStart(void)
+{
+    AppEvent event;
+    event.Type    = AppEvent::kEventType_StartSMPAdvertising;
+    event.Handler = StartSMPBLEAdvertisingHandler;
```

Comment:
```suggestion
    event.Handler = [](AppEvent*) { GetDFUOverSMP().StartBLEAdvertising(); };
```
`StartSMPBLEAdvertisingHandler` wouldn't be needed then.

Reply:
Very nice, done